.github/workflows/update-submodules.yml

```diff
@@ -0,0 +1,42 @@
+name: Submodule update on new version
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  submodule_update:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          ref: master
+          submodules: true
+          token: ${{ secrets.PAT }}
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
+
+      - name: Find version difference
+        run: |
+          cd ./packages/eth-sdk
+          npx -p npm-check-updates ncu --filter "@typechain/ethers-v5, typechain"
+          npm install
```

Comment:
we use yarn in this project. I think you need to ensure that `npm-check-updates` uses yarn under the hood too.